src/py3s_default_units.py

```diff
@@ -296,6 +298,73 @@ def format(self, output):
                 .format(braces[0], pct_str, braces[1], rem_string, status_string))
 
 
+class PY3Wireless(PY3Unit):
+    """Provide wireless network information.
+
+    Output API:
+        's_status': up, down, none
+        's_SSID'
```

Comment:
be more descriptive: SSID of what? the currently connected-to network? one with the strongest signal? etc.